CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 * 7.10.2 as default version.
 
+
+| PR | Author | Title |
+| --- | --- | --- |
+| [#753](https://github.com/elastic/ansible-elasticsearch/pull/753) | [@jmlrt](https://github.com/jmlrt) | Fix java install path when system has multiple java  |
```

Comment:
Do you know why https://github.com/elastic/ansible-elasticsearch/commit/470eae90ede5af4fbe9c501707d9d68b9abb0115 is missing here?